h2o-hadoop/h2o-mapr5.0/build.gradle

```diff
@@ -4,3 +4,10 @@ ext {
 }
 
 apply from: '../driverjar.gradle'
+
+dependencies {
+  compile (project(":h2o-jetty-8")) {
+    exclude module: "servlet-api"
```

Comment:
btw: why this part is repeated cross the build files, you can easile reuse that by having a separated `apply from: ...`

Reply:
Looks like matter of taste, but IMHO important things need to be explicit instead of reusing couple of lines.
And *dependencies are superimportant* part of build files. In most cases, they are the primary reason for looking into a buildfile. If they are not present explicitly, and are instead hidden behind some cryptocode, it costs time and frustration, and causes ugly mistakes that are not always easy to track down.

Besides, the point of adding dependency on jetty module is, they are *not always the same*. Right now, most depend on V8, only CDH6 depends on V9 of jetty. Those needing V9 will probably be coming more frequently. It's better to maintain specific dependencies, than exceptions in some magic crypto buildo script.

With multiple variants of dependencies, such reuse would mean either having multiple `driverjar.gradle` files for various sets of dependencies, or introducing some disgusting "if" branches in the existing one.
In this situation I find reuse really harmful, not just pointless.